python/src/trezorlib/cli/cardano.py

```diff
@@ -59,14 +60,46 @@ def sign_tx(client, file, protocol_magic):
 
 
 @cli.command()
-@click.option("-n", "--address", required=True, help=PATH_HELP)
+@click.option("-n", "--path-str", required=True, help=PATH_HELP)
 @click.option("-d", "--show-display", is_flag=True)
-@click.option("-p", "--protocol-magic", type=int, default=1)
+@click.option(
+    "-t", "--address-type", type=int, default=messages.CardanoAddressType.BASE
+)
+@click.option("-s", "--staking-key-path-str", type=str, default=None)
```

Comment:
call this `staking-address` ?

Reply:
Done [here](https://github.com/trezor/trezor-firmware/pull/1112/commits/aa4c7374f5a2985426319d47fe36b58392e7e6f9).